tools/releases/dockerfiles/kuma-init.Dockerfile

```diff
@@ -1,11 +1,6 @@
-# until there is a distroless iptables image we have to use something else
-FROM ubuntu:jammy-20240808@sha256:adbb90115a21969d2fe6fa7f9af4253e16d45f8d4c1e930182610c4731962658
+FROM gcr.io/k8s-staging-build-image/distroless-iptables:v0.6.4
 ARG ARCH
 
-RUN apt-get update && \
```

Comment:
So we don't need to install these packages anymore, I guess the CI issue which relies on `ubuntu-22.04` may also be resolved.

Reply:
That could be another benefit, yeah.